cmd/gcrane/cmd/gct.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/go-containerregistry/pkg/gcrane"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/google"
+	"github.com/spf13/cobra"
+)
+
+// NewCmdGct creates a new cobra.Command for the gct subcommand.
+func NewCmdGct() *cobra.Command {
+	recursive := false
+	cmd := &cobra.Command{
+		Use:   "gct",
```

Comment:
Mostly a drive-by comment, but I don't think `gct` is a very descriptive name for this command. It may also be worth explaining in the help text how this differs from `gcrane ls`.